docs/rdataretriever.rst

```diff
@@ -25,12 +25,219 @@ To install the development version, use ``devtools``
+
+- ``debug``  (Bool): Setting TRUE helps in debugging in case of errors.
+
+**Example** :
+
+::
+
+  rdataretriever :: download("iris","/Users/username/Desktop")
+
+Installation functions
+^^^^^^^^^^^^^^^^^^^^^^
+Format specific installation
+----------------------------
+**Description** : ``rdataretriever`` supports installation of datasets in three file formats through different functions: 
+
+- csv (``install_csv``)
+- json (``install_json``)
+- xml (``install_xml``)
+
+**Arguments** : These functions require same arguments.
+
+- ``dataset`` (String): Name of the dataset to install.
+
+- ``table_name`` (String): Specify the table name to install.
+
+- ``debug`` (Bool): Setting TRUE helps in debugging in case of errors.
+
+- ``use_cache`` (Bool): Setting FALSE reinstalls scripts even if they are already installed.
+
+**Example** :
+
+::
+
+  rdataretriever :: install_csv("bird-size",table_name = "Bird_Size",debug = TRUE)
+
+Database specific installation
+------------------------------
+**Description** : ``rdataretriever`` supports installation of datasets in four different databses through different functions:
+
+- MySQL (``install_mysql``)
+- PostgreSQL (``install_postgres``)
+- SQLite (``install_sqlite``)
+- MSAccess (``install_msaccess``)
+
+**Arguments for PostgreSQL and MySQL** :
+
+- ``database_name`` (String): Specify database name.
+
+- ``debug``           (Bool): Setting True helps in debugging in case of errors.
+
+- ``host``          (String): Specify host name for database.
+
+- ``password``      (String): Specify password for database.
+
+- ``port``             (Int): Specify the port number for installation.
+
+- ``quiet``           (Bool): Setting True minimizes the console output.
+
+- ``table_name``    (String): Specify the table name to install.
+
+- ``use_cache``       (Bool): Setting False reinstalls scripts even if they are already installed.
+
+- ``user``          (String): Specify the username.
+
+**Example** :
+
+::
+
+  rdataretriever :: install_postgres(dataset = 'portal', user='postgres', password='abcdef')
+
+**Arguments for MSAccess and SQLite** :
+
+- ``file`` (String): Enter file_name for database.
+
+- ``table_name `` (String): Specify the table name to install.
+
+- ``debug`` (Bool): Setting True helps in debugging in case of errors.
+
+- ``use_cache`` (Bool): Setting False reinstalls scripts even if they are already installed.
+
+**Example** :
+
+::
+
+  rdataretriever :: install_sqlite(dataset = 'iris', file = 'sqlite.db',debug=FALSE, use_cache=TRUE)
+
+get_updates()
+^^^^^^^^^^^^^
+**Description** : This function will check if the version of the retriever’s scripts in your local directory ‘
+~/.retriever/scripts/' is up-to-date with the most recent official retriever release.
+
+**Example** :
+
+::
+
+  rdataretriever :: get_updates()
+
+reset()
+^^^^^^^
+**Description** : The function will Reset the components of rdataretriever using scope [ all, scripts, data, connection]
```

Comment:
We've dropped the storing of connection information in the core retriever. If we're still doing so in `rdataretriever` this is fine. If we've dropped it there as well then we should drop references to `connection`.